src/PresentationalComponents/Inventory/helpers.js

```diff
@@ -69,13 +69,11 @@ export const getEntities =
       selectedTags,
     } = config;
 
-    //operating_system is currently not supported, but will be down the line.
-    const sort =
-      orderBy === 'operating_system'
-        ? 'rhel_version'
-        : `${orderDirection === 'ASC' ? '' : '-'}${
-            orderBy === 'updated' ? 'last_seen' : orderBy
-          }`;
+    const sort = `${orderDirection === 'ASC' ? '' : '-'}${
```

Comment:
Can we extract this into a utility function so that it can be reused and clean this function a bit?

Reply:
Good idea.  Done!